Gemfile

```diff
@@ -54,14 +53,20 @@ gem 'jquery-ui-rails'
 # CORS
 gem 'rack-cors', :require => 'rack/cors'
 
-gem 'rainbow'
 
 # Use ActiveModel has_secure_password
 gem 'ruby-saml'
 
 gem 'strong_migrations'
 gem 'will_paginate'
 
+group :production do
+  # Use SCSS for stylesheets
+  gem 'sass-rails', '~> 5.0'
+
+  gem 'rainbow'
```

Comment:
Since `rainbow` is used explicitly in the `lint` and `security` rake tasks (which are run as part of CI), let's move this to the global namespace/group.

I know it gets included by `rubocop` but we should still explicitly list it for all environments.
Removing `rubocop` should not break those rake tasks, which would happen without `rainbows` included  globally (not that we plan to ever remove `rubocop`)

Reply:
Moved